src/dummy_audio.rs

```diff
@@ -0,0 +1,17 @@
+use projectm_rs::core::{projectm, projectm_handle};
+
+pub fn generate_random_audio_data(projectm_handle: projectm_handle) {
```

Comment:
can this be `fn` or `pub(crate) fn`?

Reply:
Yeah, you want the smallest scope. I'll pull in this pull request on my machines (Windows, Mac, and probably Linux) and then try it out on each to make sure my suggestions make sense.